### PR TITLE
[docs] [integrations] Docs for Apache Atlas

### PR DESCRIPTION
**Pull request by aishwarya24**
build preview :https://deploy-preview-16549--infallible-bardeen-164bc9.netlify.app/preview/integrations/atlas-ycql/

### CODE REVIEW

File: docs/content/preview/integrations/atlas-ycql.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+This tutorial describes how to set up Apache Atlas with YugabyteDB and run the quick start provided by the Atlas service.
+
+## Prerequisites
+
+To use [Apache Atlas](https://doc.akka.io/docs/akka-persistence-r2dbc/current/overview.html), ensure that you have the following:
+
+- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../quick-start/).
+
+- [Apache Solr 5.5.1](https://solr.apache.org/guide/6_6/installing-solr.html) is installed.
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:23]:
```suggestion
- [Apache Solr 5.5.1](https://solr.apache.org/guide/6_6/installing-solr.html) installed.
```

File: docs/content/preview/integrations/atlas-ycql.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+## Build the Apache Atlas Project
+
+To get the Apache Atlas server file, you need to build the Apache Atlas source using the following steps:
+
+1. Clone the [source](https://github.com/apache/atlas) from GitHub to your local setup. Checkout checkout to the latest stable release tag ( For example: release-2.3.0), and follow the steps in the README to build the files.
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:29]:
```suggestion
1. Clone the [source](https://github.com/apache/atlas) from GitHub to your local setup. Checkout to the latest stable release tag (for example, release-2.3.0) and follow the steps in the README to build the files.
```

File: docs/content/preview/integrations/atlas-ycql.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    atlas.graph.storage.port=9042
+    atlas.EntityAuditRepository.impl=org.apache.atlas.repository.audit.CassandraBasedAuditRepository
+
+    #Comment the following Hbase specific storage properties properties
+    #Hbase
+    #atlas.graph.storage.hostname=localhost
+    #atlas.graph.storage.hbase.regions-per-server=1
+    ```
+
+1. Change the `atlas.graph.index.search.solr.zookeeper-url` in the `conf/atlas-application.properties` file to point to zookeeper started by Solr. The default value is: `localhost:9983`. An example Solr URL is as follows:
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:72]:
```suggestion
1. Change the `atlas.graph.index.search.solr.zookeeper-url` in the `conf/atlas-application.properties` file to point to ZooKeeper started by Solr. The default value is `localhost:9983`. An example Solr URL is as follows:
```

File: docs/content/preview/integrations/atlas-ycql.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    {"Description":"Metadata Management and Data Governance Platform over Hadoop",    "Revision":"4cd215e1e2a04acbcd8afe6af95f43c4979202f1","Version":"2.3.0","Name":"apache-atlas"}
+    ```
+
+1. Run the quick start script using the following command:
+
+    ```sh
+    bin/quick_start.py
+    ```
+
+    When prompted for a username and password, enter `admin`. After it completes, you should see the following output:
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:114]:
```suggestion
    When prompted for a username and password, enter `admin`. After the script completes, you should see the following output:
```

File: docs/content/preview/integrations/atlas-ycql.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    loadSalesDaily(LoadProcess) -> sales_fact_daily_mv(Table)
+    Sample data added to Apache Atlas Server.
+    ```
+
+1. You can also verify if the keyspaces `janusgraph` and `atlas_audit` got created using the [ycqlsh](../../admin/ysqlsh/#starting-ysqlsh) shell as follows:
+
+    ```output
+    DESC KEYSPACES;
+    ```
+
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:152]: any output?

File: docs/content/preview/integrations/atlas-ycql.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+1. Stop the Atlas server using the following command:
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:153]: Would put this cleanup in a separate section. Don't need numbered steps
## Clean up
You can stop the Atlas server using the following command:
etc

File: docs/content/preview/integrations/atlas-ycql.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+menu:
+  preview_integrations:
+    identifier: ycql-atlas
+    parent: integrations
+    weight: 571
+type: docs
+---
+[Apache Atlas](https://atlas.apache.org/) is an enterprise-scale open data management service which provides governance for Hadoop and the entire enterprise data ecosystem.
+
+[Solr](https://solr.apache.org/) is an open-source indexing platform that serves as an indexing backend to run Apache Atlas.
```
Comment (Sfurti-yb) [inline docs/content/preview/integrations/atlas-ycql.md:14]: Can this line be moved to the prerequisites section where we mention Solr?

**Review by Sfurti-yb — APPROVED** [review]
LGTM. Please just not the comment below.

File: docs/content/preview/integrations/atlas-ycql.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+    Sample Lineage Info:
+    loadSalesMonthly(LoadProcess) -> sales_fact_monthly_mv(Table)
+    time_dim(Table) -> loadSalesDaily(LoadProcess)
+    sales_fact_daily_mv(Table) -> loadSalesMonthly(LoadProcess)
+    sales_fact(Table) -> loadSalesDaily(LoadProcess)
+    loadSalesDaily(LoadProcess) -> sales_fact_daily_mv(Table)
+    Sample data added to Apache Atlas Server.
+    ```
+
+1. You can also verify if the keyspaces `janusgraph` and `atlas_audit` got created using the [ycqlsh](../../admin/ysqlsh/#starting-ysqlsh) shell as follows:
```
Comment (ddhodge) [inline docs/content/preview/integrations/atlas-ycql.md:146]:
```suggestion
1. You can verify that the `janusgraph` and `atlas_audit` keyspaces were created using the [ycqlsh](../../admin/ysqlsh/#starting-ysqlsh) shell as follows:
```

Comment (Sfurti-yb) [reply docs/content/preview/integrations/atlas-ycql.md:146]: Just noticed, the ycqlsh link is pointing to the ysqlsh page

**Pull request merged**